tools/utils.ts

```diff
@@ -183,9 +183,13 @@ export function hasSchemaFlag<T, K extends keyof T>(schema: T, flag: K): schema
   return Boolean(schema[flag]);
 }
 
+export function isPackageVersionConverged(version: string) {
+  return version.startsWith('9.');
+}
```

Comment:
Once this will start with a caret it will be no longer safe, do you want to change this?